build.cmd

```diff
@@ -67,7 +67,7 @@ set "__IntermediatesDir=%__RootBinDir%\obj\%__BuildOS%.%__BuildArch%.%__BuildTyp
 set "__PackagesBinDir=%__BinDir%\.nuget"
 set "__TestBinDir=%__RootBinDir%\tests\%__BuildOS%.%__BuildArch%.%__BuildType%"
 set "__TestIntermediatesDir=%__RootBinDir%\tests\obj\%__BuildOS%.%__BuildArch%.%__BuildType%"
-set "__GeneratedIntermediatesDir=%__IntermediatesDir%\Generated"
+set "__GeneratedIntermediatesDir=%__IntermediatesDir%\Generated_present" ::use this variable to locate dynamically generated files, the actual location though will be different
```

Comment:
The idea is to make this mechanism common to all generated files, though it is applicable only for eventing today. so I will change Generated_present to Generated_latest, sounds good ?